src/main/resources/migrations/00040_create_table_payouts.sql

```diff
@@ -0,0 +1,19 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:create_table_payouts
+CREATE TABLE payouts (
+    id BIGSERIAL PRIMARY KEY,
+    gateway_external_id VARCHAR(26),
```

Comment:
anything that has `external_id` may mix up with our terminology. May be gateway_payout_id or something similar ?